huxley/api/views/register.py

```diff
@@ -50,6 +59,59 @@ def register(self, request, *args, **kwargs):
             registration_serializer.is_valid(raise_exception=True)
             registration_serializer.save()
 
+        if Conference.get_current().invoicing_enabled and handler is not None:
```

Comment:
Why will `handler` ever be `None`?

Reply:
If authentication fails, the exception is handled by setting `handler` to `None`. This also allows the existing unit test suite to not fail, since we won't try to make QuickBooks API calls with an unauthenticated handler